extend.php

```diff
@@ -39,14 +39,17 @@
         ->type(UserSuspendedBlueprint::class, BasicUserSerializer::class, ['alert', 'email'])
         ->type(UserUnsuspendedBlueprint::class, BasicUserSerializer::class, ['alert', 'email']),
 
-    function (Dispatcher $events) {
-        $events->subscribe(Listener\AddUserSuspendAttributes::class);
-        $events->subscribe(Listener\RevokeAccessFromSuspendedUsers::class);
+    (new Extend\Event())
```

Comment:
Do we need / want to create whole classes for these new listeners? They seem small / simple enough that they could just be closures, and don't need any DI

Reply:
I personally do like using invokable classes instead, I'm not a big fan of having logic code in the extend.php file.